apps/charterafrica/README.md

```diff
@@ -16,7 +16,22 @@ and modify the `.env.local` file according to your needs.
 
 The default `.env` file is for the publicly visible environment variables.
 **DO NOT** include any secrets in it. All secrets should go into `env.local`.
-For more, see <<nextjs env var docs>>
+For more, see NextJS env var docs [here](https://nextjs.org/docs/basic-features/environment-variables).
+
+## Database setup
+
+Generate a new MongoDB keyfile by running the following command:
+
+```bash
+openssl rand -base64 741 > ./mongo-keyfile
+chmod 600 ./mongo-keyfile
+```
```

Comment:
Can't this be included in the Makefile? May be under `mongodb-keyfle`

Reply:
1. If we have moved this to Makefile, shouldn't this doc be updated?
2. Shouldn't this command be run in the project root directory rather than app directory?